### DISCUSSION TITLE
How to access the user's timeline

### DISCUSSION THREAD

**Discussion by hemal08ce094** [sbertix/Swiftagram#132] [Q&A] [answered]
i went through the project i am not getting how to load the user's daily feed or maybe its named differently,

is it  explore(startingAt ? which Model it will map to.   Can you please guide with some real-time code examples?

**Comment by sbertix** [comment, marked as answer]
Hey @hemal08ce094,

Are you referring to a user's timeline or their posts? 
You can get the timeline with `Endpoint.Media.Posts.timeline()` and a user's post with `Endpoint.Media.Posts.owned(by:)`. 
Hope it helps :blush:

**Reply by hemal08ce094** [reply]
Hey @sbertix  you got it right, I was looking for this `Endpoint.Media.Posts.timeline()`

btw how to map its response model. ? 
it has so many values, looks so bit challenging to parse them manually as I am trying to map it on WatchOS.

**Reply by hemal08ce094** [reply]
What I want from that model is posted by userName, Date/time, Media content.

**Reply by sbertix** [reply]
It's complicated, you're right, and it's the reason why there's not an actual model for it, as it also changed a lot lately. 

Given a successful response `Wrapper` (`response`), you can obtain all unwatched posts in `response.feedItems().array()`, all suggested posts and already watched posts in `response.feedItems().array()?.last?.endOfFeedDemarcator.groupSet.groups.array()` and again inside these groups you'll find a `feedItems` array.  
`feedItems` arrays contain dictionary items, and you can access the properties you need by referring to their `mediaOrAd` property (which is of course a `Wrapper`).
